cors/304.htm

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>CORS - 304 Responses</title>
+<meta name=author title="Mark Nottingham" href="mailto:mnot@mnot.net">
+
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=support.js?pipe=sub></script>
+
+<h1>CORS - 304 Responses</h1>
+<div id=log></div>
+<script>
+
+
+/*
+ * 304 Responses
+ */
+
+// A header used to correlate requests and responses
+var state_header = "content-language"
+
+function req(url, id, t, ready) {
+  console.log('starting request ' + id)
+  var client = new XMLHttpRequest()
+  client.error_caught = false
+  try {
```

Comment:
Asynchronous XMLHttpRequest does not throw (unless you do method invocation incorrectly) so `try` is not needed.